src/PointwiseFunctions/AnalyticData/GrMhd/OrszagTangVortex.hpp

```diff
@@ -0,0 +1,170 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Minkowski.hpp"
+// IWYU pragma: no_include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"  // for IdealFluid
+#include "PointwiseFunctions/Hydro/EquationsOfState/IdealFluid.hpp"  // IWYU pragma: keep
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "Utilities/Requires.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace grmhd {
+namespace AnalyticData {
+
+/*!
+ * \brief Analytic initial data for the relativistic Orszag-Tang vortex.
+ *
+ * The relativistic version of the Orszag-Tang vortex, described, for
+ * example, in [arXiv:1101.3573](https://arxiv.org/abs/1101.3573), is
```

Comment:
Once we have a 25MB bib file that makes editing it take a long time you may change your mind on if you want it, but I don't disagree that having all references in one place is nice in theory.

I think if the open access links are properly displayed by Doxygen (by which I mean they are actually displayed. Some journals have bibs that don't show e-print links) and we add some documentation on how to do all this (probably [here](https://spectre-code.org/writing_good_dox.html)), it would be a nice solution.

The hovering over sounds like a nice feature :)

Reply:
@nilsdeppe @nilsleiffischer If you want to switch to the arXiv-version easily you could also use the browser extension
http://unpaywall.org/products/extension

But in general I agree that having the arXiv version in the first place should be preferred.

Reply:
![bildschirmfoto 2018-11-27 um 16 32 15](https://user-images.githubusercontent.com/746230/49092311-12bea180-f262-11e8-98dc-54f6be9b98a0.png)
This is how Doxygen displays the bib entry by default. Not sure how to change that, could be possible. That's out of scope of this PR, but we should discuss these things in issues.

Reply:
#1201

Reply:
That's a cool extension, @hrueter! I'll need to use that :)

Reply:
So, what is the suggestion for what I actually do here?  I agree with @nilsdeppe that the arXiv links need to be prominently available, so I'm leaning towards reverting to the direct arXiv link until the reference stuff can be sorted out.

Reply:
I'm in favor of that and then we can do a global search for arxiv links later when we know how to replace them properly :)

Reply:
Or: we keep both the reference and the arXiv link for now, then later remove the arXiv links when we have made Doxygen show them automatically. So this would look like

> ... as described in [1] ([arXiv:1101.3573](https://arxiv.org/abs/1101.3573))

I see you have pushed a fixup already. The time zone differences sometimes delay replies by ~ half a day, please understand. But my suggestion would be easy to implement by dropping the last fixup commit and adding the arXiv links instead.

Reply:
As long as the arxiv links are present I'm in favor of whatever gets the PR in quickest :)